client/blocks/eligibility-warnings/warning-list.tsx

```diff
@@ -121,25 +124,16 @@ function getWarningDescription(
 			);
 
 		case 'hosting-features':
-			return hasTranslation(
-				'By activating all hosting features the following change will be made to the site:'
-			)
-				? translate(
-						'By activating all hosting features the following change will be made to the site:',
-						'By activating all hosting features the following changes will be made to the site:',
-						{
-							count: warningCount,
-							args: warningCount,
-						}
-				  )
-				: translate(
-						'By activating all developer tools the following change will be made to the site:',
-						'By activating all developer tools the following changes will be made to the site:',
-						{
-							count: warningCount,
-							args: warningCount,
-						}
-				  );
+			return preventWidows(
```

Comment:
I removed the `hasTranslation` check because the i18n check says that all strings are translated. I also added widow prevention.